Components/ListOfCharts/index.js

```diff
@@ -3,20 +3,25 @@ import { Container } from "../Container";
 import ChartItem from "../ChartItem";
 import { chartArray } from "../../lib/listOfPlotTypes";
 
-export default function ListOfCharts({ onSelectChartType, keynames }) {
+export default function ListOfCharts({
+  onSelectChartType,
+  keynames,
+  clickedChartType,
+}) {
   return (
     <>
       {keynames.length > 0 && (
         <>
           <Heading>Step 2: Select the type of chart</Heading>
           <Container $wrap="wrap">
-            {chartArray.map((chart) => {
+            {chartArray.map(({ name, icon }) => {
               return (
                 <ChartItem
-                  key={chart.name}
-                  icon={chart.icon}
-                  name={chart.name}
-                  onSelectChartType={onSelectChartType}
+                  key={name}
```

Comment:
Do the entries of your chartArray also have an id? If so, I would prefer seeing the id as the key.

Otherwise, the name serves fine

Reply:
No, we don't have an id.